### DISCUSSION TITLE
Tremblant not behaving

### DISCUSSION THREAD

**Discussion by andedl** [GrandOrgue/grandorgue#974] [Help]
Hi all, 

I am using the standard organ in GO 3.4.3-1, with an Motu M2 audio card and a midi pedalboard connected.
All of a sudden, the tremblant will start. I turn it off, and after a while it goes on again. Sometimes I cannot even turn it off.
What should I try?

Thanks
Andreas

**Comment by larspalo** [comment]
Check what MIDI messages are sent/received with a MIDI monitor, and what MIDI events are set for the tremulant in question. I'd bet that some MIDI message (that's un-intended - but still sent by the hardware) is triggering (activating) the tremulant. It's fairly common that if the MIDI equipment is in a poor working state, malfunctioning or simply mis-configured - it can send rather random (or not) messages.

**Reply by andedl** [reply]
Thanks Lars for the pointer, I'll try that.
Now I tried to connect the midi pedalboard directly to a digital piano - no problem. So maybe the problem is in the sound card with the midi interface to the computer.
It seems that the problem arises whenever I have run my notation program that sort of "claims" the sound card in a slightly obtrusive way, so for now restarting the sound card solves the problem.
